posts/automated-api-testing-with-postman.md

```diff
@@ -0,0 +1,169 @@
+---
+authors:
+- Matthew Reed
+- Amine Ouali Alami
+tags:
+- API
+- TDD
+- Testing
+- Postman
+
+date: 2020-09-26T20:16:10.000Z
+title: "Start Automating API Tests with Postman"
+image: https://raw.githubusercontent.com/ippontech/blog-usa/master/images/2020/10/postman_logo.png
+---
+
+## What is Wrong with Cucumber
+
+In short, nothing is wrong with Cucumber. For a long time now many of us have been productively using Cucumber or one of its variations for Acceptance Test Driven Development (ATDD). The English-written tests are the gates to our applications that keep breaking changes out of the codebase. This approach of bridging the business and development sides is going to continue even after this blog. Yet, hardly every Product Owner is actively writing and maintaining these tests on their projects. Only the high-level test results come up anyway when the CI/CD pipeline runs, so developers are often needed to investigate no matter what.
+
+Why does there need to be such a syntactically complex system in place for a group that looks at the feature file only every few months?
+
+Well, good news everyone! There is another way to write integration tests already built into the common development testing tool, [Postman](https://www.guru99.com/postman-tutorial.html). It is simple to get started and, as will be shown, they can be easily integrated with [Newman](https://learning.postman.com/docs/running-collections/using-newman-cli/command-line-integration-with-newman/) to run automatically within a CI/CD pipeline. Soon you will wonder if Cucumber is necessary for your project, and may even choose to forget it altogether.
```

Comment:
May also want to define here or in a preceding paragraph what Postman and Newman do at a high level.

Reply:
Hmm I'll take a look and try to improve based on your suggestion